packages/eas-cli/src/project/ios/entitlements.ts

```diff
@@ -28,38 +28,43 @@ export async function getManagedApplicationTargetEntitlementsAsync(
       ...env,
     };
 
-    let expWithMods: ExportedConfig;
+    const projectExpoPrebuildConfigPath = path.join(
+      projectDir,
+      'node_modules',
+      '@expo',
+      'prebuild-config'
+    );
+    const projectExpoConfigPluginsPath = path.join(
+      projectDir,
+      'node_modules',
+      '@expo',
+      'config-plugins'
+    );
```

Comment:
This won't work for monorepos, we can't hardcode paths towards `node_modules/..`. Use `require.resolve` instead.